# llvm-17.0.1.src/lib/ExecutionEngine/RuntimeDyld/RuntimeDyldELF.cpp:2187:51: error: cannot convert 'llvm::Align' to 'uintptr_t' {aka 'long unsigned int'} in initialization
# 2187 |         uintptr_t StubAlignment = getStubAlignment();
#      |                                   ~~~~~~~~~~~~~~~~^~
#      |                                                   |
#      |                                                   llvm::Align
#llvm-17.0.1.src/lib/ExecutionEngine/RuntimeDyld/RuntimeDyldELF.cpp:2187:19: warning: unused variable 'StubAlignment' [-Wunused-variable]
# 2187 |         uintptr_t StubAlignment = getStubAlignment();
# it seems that StubAlignment is not used at all so just remove

--- llvm-18.1.8.src/lib/ExecutionEngine/RuntimeDyld/RuntimeDyldELF.cpp.orig
+++ llvm-18.1.8.src/lib/ExecutionEngine/RuntimeDyld/RuntimeDyldELF.cpp
@@ -2185,7 +2185,7 @@
 
         uintptr_t BaseAddress =
           reinterpret_cast<uintptr_t>(Section.getAddress());
-        uintptr_t StubAlignment = getStubAlignment();
+//        uintptr_t StubAlignment = getStubAlignment();
         StubAddress = BaseAddress + StubOffset;
         StubAddress = llvm::alignTo(StubAddress, 4U);
         StubOffset = StubAddress - BaseAddress;
@@ -2216,7 +2216,7 @@
     } else {
       uintptr_t BaseAddress =
         reinterpret_cast<uintptr_t>(Section.getAddress());
-      uintptr_t StubAlignment = getStubAlignment();
+//      uintptr_t StubAlignment = getStubAlignment();
       StubAddress = BaseAddress + StubOffset;
       StubAddress = llvm::alignTo(StubAddress, 4U);
       StubOffset = StubAddress - BaseAddress;
@@ -2257,7 +2257,7 @@
 
         uintptr_t BaseAddress =
           reinterpret_cast<uintptr_t>(Section.getAddress());
-        uintptr_t StubAlignment = getStubAlignment();
+//        uintptr_t StubAlignment = getStubAlignment();
 
         StubAddress = BaseAddress + StubOffset;
         StubAddress = llvm::alignTo(StubAddress, 8U);
@@ -2299,7 +2299,7 @@
     } else {
       uintptr_t BaseAddress =
         reinterpret_cast<uintptr_t>(Section.getAddress());
-      uintptr_t StubAlignment = getStubAlignment();
+//      uintptr_t StubAlignment = getStubAlignment();
       StubAddress = BaseAddress + StubOffset;
       StubAddress = llvm::alignTo(StubAddress, 8U);
       StubOffset = StubAddress - BaseAddress;
